addons/common/functions/fnc_moveCursor.sqf

```diff
@@ -0,0 +1,30 @@
+/*
+ * Author: LorenLuke (function), BaerMitUmlaut (Extension)
+ * Simulates mouse input by number of pixels (not consistent/smooth)
```

Comment:
Optional animated movement? linear or even specific curves?

Reply:
Rotating a foot-mobile is all in one go, but vehicles had some sort of acceleration and cap on how fast you can traverse, meaning it has to be inputting for more than one frame to do much of anything.

Reply:
What is a  `foot-mobile`?

Reply:
A person. Don't you play proper Arma, that's standard callout. :p

Reply:
I intended this to mean adding animation handled in a second thread.
Which also solves:
>meaning it has to be inputting for more than one frame to do much of anything.